src/server/api/blockallowlist.ts

```diff
@@ -45,7 +58,8 @@ export const blockAllowListRoutes = (cfg: APIConfig, store: Store, apsystem: Act
       return await reply.code(403).send('Not Allowed')
     }
 
-    const accounts = request.body.split('\n')
+    const accounts = request.body.trim().split('\n')
```

Comment:
maybe `.map(x => x.trim())` after the split? also remove empty lines

Reply:
I think empty lines should be an error.